src/Symfony/Component/Notifier/Bridge/Smsapi/README.md

```diff
@@ -0,0 +1,4 @@
+SMSAPI Notifier
+===============
+
+Provides Smsapi integration for Symfony Notifier.
```

Comment:
Can you add the DSN pattern as well? And the links at the end like done for the other norifiers?